CONTRIBUTING.md

```diff
@@ -1,106 +1,77 @@
-- **Suggest features!** Have an idea for something mathsteps should solve or a way
-  for it to teach math better? If your idea is not an [existing issue](https://github.com/socraticorg/mathsteps/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement), create a new issue with
-  the label "enhancement"
-- **Report bugs!** If the bug is not an [existing issue](https://github.com/socraticorg/mathsteps/issues?q=is%3Aopen+is%3Aissue+label%3Abug),
-  create a new issue with the label "bug" and as much detail as you can so
-  someone else can reproduce it.
-- **Contribute code!** We'd love to have more contributors working on this. Check
-  out the section below with more information on how to contribute, and feel
-  free to email mathsteps@socratic.org with any questions!
+- **Spread the word!** If you think mathsteps is cool, tell your friends! Let them know they can use this and that they can contribute.
+- **Suggest features!** Have an idea for something mathsteps should solve or a way for it to teach math better? If your idea is not an [existing issue](https://github.com/socraticorg/mathsteps/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement), create a new issue with the label "enhancement".
+- **Report bugs!** If the bug is not an [existing issue](https://github.com/socraticorg/mathsteps/issues?q=is%3Aopen+is%3Aissue+label%3Abug), create a new issue with the label "bug" and provide as many details as you can, so that someone else can reproduce it.
+- **Contribute code!** We'd love to have more contributors working on this. Check out the section below with more information on how to contribute, and feel free to mail us at mathsteps@socratic.org with any questions!
 
 ### Creating a pull request
 
-- we're excited to see your [pull request](https://help.github.com/articles/about-pull-requests/)!
-- if you want to work on something, please comment on the
-  [related issue](https://github.com/socraticorg/mathsteps/issues) on GitHub
-  before you get started, so others are aware you're working on it
-- if there is no existing issue for the change you'd like to make, you can make
-  a [new one](https://github.com/socraticorg/mathsteps/issues/new)
-- make sure all the unit tests pass (with `npm test`) before creating the
-  pull request, and please add your own tests for what you changed as well.
-  If you're not sure how to add tests or are confused about why tests are
-  failing, it's also fine to create the pull request first and we'll help
-  you get things working.
+We're excited to see your [pull request](https://help.github.com/articles/about-pull-requests/)!
 
-### Testing
+- If you want to work on something, please comment on the [related issue](https://github.com/socraticorg/mathsteps/issues) on GitHub before you get started, so others are aware that you're working on it. In case there's is no existing issue for the change you'd like to make, you can [create a new issue](https://github.com/socraticorg/mathsteps/issues/new).
```

Comment:
"In case there's" -> "In the case that there's"

(or "If there's no existing...")